api/app/models/project.rb

```diff
@@ -135,6 +117,12 @@ def resource_kinds
       .to_a.pluck(:kind)
   end
 
+  def resource_tags
```

Comment:
Hmmm.... I don't love this approach, as it introduces what could potentially be a hefty query for every single project serialization. It's fine for now, but I wonder if we shouldn't instead do a separate request for tags by project when we need that.